scaleservice/service.proto

```diff
@@ -1,40 +1,17 @@
 syntax = "proto3";
 package pb;
 
-
+option go_package = "./pb";
```

Comment:
why do we have to add this line since line 2 already have pb declaration ?
Can we remove that ?

Reply:
In my machine, can't run without that line bro. If I remove that, I get the following error message.

-----------

Generating Proto files for /home/atzm/Documents/projects/sheet-pilot/sheet-pilot-proto/scaleservice....
protoc-gen-go: unable to determine Go import path for "service.proto"

Please specify either:
        • a "go_package" option in the .proto source file, or
        • a "M" argument on the command line.

See https://developers.google.com/protocol-buffers/docs/reference/go-generated#package for more information.

--go_out: protoc-gen-go: Plugin failed with status code 1.